homeassistant/helpers/config_validation.py

```diff
@@ -1690,6 +1691,10 @@ def _base_trigger_validator(value: Any) -> Any:
         **SCRIPT_ACTION_BASE_SCHEMA,
         vol.Required(CONF_STOP): vol.Any(None, string),
         vol.Optional(CONF_ERROR, default=False): boolean,
+        vol.Optional(CONF_RESPONSE): vol.Any(
+            vol.All(dict, template_complex),
+            vol.All(str, template),
```

Comment:
Explicitly allowing a string template because `template_complex` will not wrap a static string in a `Template` class and thus not parsing it on render.